app/javascript/gobierto_visualizations/webapp/containers/contracts/ContractsShow.vue

```diff
@@ -132,6 +237,40 @@ export default {
       this.contract_type = contract_type
       this.permalink = permalink
       this.assignee_routing_id = assignee_routing_id
+      this.contractor = contractor
+      this.contract_type = contract_type
+      this.start_date = start_date
+      this.end_date = end_date
+      this.award_date = award_date
+      this.batch_number = +batch_number
+      this.minor_contract = minor_contract
+      this.open_proposals_date = open_proposals_date || null
+      this.submission_date = submission_date || null
+      this.cpvs = cpvs
+      this.category_title = category_title
+    }
+
+    if (this.hasBatch) this.groupBatchs()
+  },
+  methods: {
+    groupBatchs() {
+      this.filterContractsBatchs = this.contractsData.filter(({ title }) => title === this.title).sort((a, b) => a.batch_number - b.batch_number);
+    },
+    calculatePercentage(value) {
+      const filterByContractor = this.contractsData.filter((contract) => contract[value] === this[value])
+      const totalAmount = filterByContractor.reduce((acc, { final_amount_no_taxes } ) => acc + final_amount_no_taxes, 0)
+      return ((this.final_amount_no_taxes * 100) / totalAmount).toFixed(2)
+    },
+    parseDate(value) {
+      if (!value) return
+      const convertDate = new Date(value)
+      const year = convertDate.getFullYear()
+      const day = convertDate.getDate()
+      const indexMonth = convertDate.getMonth()
+      const { [I18n.locale]: { shortMonths } } = d3locale
+      const month = shortMonths.filter((d, index) => index === indexMonth)
+
+      return `${day} ${month} ${year}`
```

Comment:
What's that? Are you just trying to format a date? We've `date` vue-filter, or as a plain function also, to do so.
In any case, doesn't `toLocaleDateString(....options)` work to you?

Reply:
Yes, I tried to convert it into a specific format(example: 22 ene 2021). In this case, `toLocaleDateString` works fine.